Sources/App/Setup/routes.swift

```diff
@@ -0,0 +1,15 @@
+import Vapor
+
+/// Register your application's routes here.
+public func routes(_ router: Router) throws {
+    // Basic "Hello, world!" example
+    router.get("hello") { _ in
+        return "Hello, world!"
+    }
+
+    // Example of configuring a controller
+    let todoController = TodoController()
```

Comment:
Instead of registering the controller one route at a time, register the entire collection like this:

```swift
try router.register(collection: TodoController())
```

(https://github.com/vapor-community/styleguide#routesswift)